pkg/minikube/cluster/cluster.go

```diff
@@ -122,12 +132,27 @@ func DeleteHost(api libmachine.API) error {
 	if err != nil {
 		return err
 	}
+
+	if err = unregister(host); err != nil {
+		return err
+	}
+
 	m := util.MultiError{}
 	m.Collect(host.Driver.Remove())
 	m.Collect(api.Remove(constants.MachineName))
 	return m.ToError()
 }
 
+// Un-register host VM
+func unregister(host *host.Host) error {
+	if drivers.MachineInState(host.Driver, state.Running)() {
```

Comment:
@hferentschik Agree, should we use config file which we create at the beginning for this purpose or create a another one?